### PR TITLE
Keep Spark version in a single place only (BeamModulePlugin)

### PR DESCRIPTION
**Pull request by mosche**
It's a bit of a trap to have the Spark version in two places.

Especially because it's most obvious to change the one in the Spark build.gradle. But that one is always overruled by the one used in the artifact library in BeamModulePlugin.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by mosche** [comment]
R: @aromanenko-dev

**Comment by aromanenko-dev** [comment]
Retest this please

**Comment by aromanenko-dev** [comment]
Run Spark StructuredStreaming ValidatesRunner

**Comment by aromanenko-dev** [comment]
Run Spark ValidatesRunner

**Comment by aromanenko-dev** [comment]
Run Java_Debezium_IO_Direct PreCommit

**Comment by aromanenko-dev** [comment]
Run Portable_Python PreCommit

File: runners/spark/2/build.gradle [hunk 1]
```diff
@@ -19,8 +19,8 @@
 def basePath = '..'
 /* All properties required for loading the Spark build script */
 project.ext {
-  // Set the version of all Spark-related dependencies here.
-  spark_version = '2.4.8'
+  // Spark 2 version as defined in BeamModulePlugin
+  spark_version = spark2_version
```
[the remark this answers is not in the document]
Comment (mosche) [reply runners/spark/2/build.gradle:23]: Not going to fix all inconsistencies at once ;)

File: buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy [hunk 2]
```diff
@@ -499,6 +499,10 @@ class BeamModulePlugin implements Plugin<Project> {
     def arrow_version = "5.0.0"
     def jmh_version = "1.34"
 
+    // Export Spark versions, so they are defined in a single place only
+    project.ext.spark2_version = spark2_version
```
[the remark this answers is not in the document]
Comment (mosche) [reply buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:503]: I'll add some comments as part of https://github.com/apache/beam/issues/23728

**Pull request merged**